### PR TITLE
[WEB] FAQ

### PR DESCRIPTION
**Pull request by calaca**
## Summary

Partial #531 

## Relevant technical choices

This issue implements only the front-end side of things to the FAQ feature.
We'll need to come back later when the API is ready.

## QA Steps

1. Open a technology details page and navigate to the FAQ tab
image
2. Login as a researcher and verify that you have the questions along with the notification counter showing at the dropdown menu
image
image
3. Navigate to the questions page and verify that all of them are being listed and can be viewed
image
image

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

**Review by mateus4k — APPROVED** [review]
LGTM 👍

File: packages/web/pages/user/my-account/questions.js [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+	const { query } = ctx;
+
+	const page = Number(query.page) || 1;
+
+	return {
+		questions: questionsMock,
+		currentPage: page,
+		totalPages: 1,
+		totalItems: 3,
+		currentSort: { by: query.orderBy, order: query.order },
```
Comment (lcnogueira) [inline packages/web/pages/user/my-account/questions.js:187]: Was this sort feature supposed to be working at the moment? As far as I can see and doesn't order the values. It changes the URL query values, but the table is not updated. Not sure it this is something related to this feature or to the DataGrid...

cc @luizeboli

File: packages/web/components/Technology/Details/Tabs/FAQ/styles.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+		}
+
+		@media (max-width: ${screens.medium}px) {
+			max-width: 100%;
+			flex-direction: column;
+			align-items: flex-start;
+
+			> div {
+				width: 100%;
+				max-width: 100%;
```
Comment (lcnogueira) [inline packages/web/components/Technology/Details/Tabs/FAQ/styles.js:20]: Is it necessary, since they are both 100%?

**Review by lcnogueira — APPROVED** [review]
Great job, Lorena! 💯 🎉

File: packages/web/pages/user/my-account/questions.js (hunk 1 above)
Comment (luizeboli) [reply packages/web/pages/user/my-account/questions.js:187]: Sadly ordering doesn't works for some routes.

Comment (lcnogueira) [reply packages/web/pages/user/my-account/questions.js:187]: Hm...got it..so it's something related to the api...

Comment (calaca) [reply packages/web/pages/user/my-account/questions.js:187]: And there's no API for this issue yet.

File: packages/web/components/Technology/Details/Tabs/FAQ/styles.js (hunk 2 above)
Comment (calaca) [reply packages/web/components/Technology/Details/Tabs/FAQ/styles.js:20]: I do, because of the desktop styles.

**Pull request merged**